c10/core/DispatchKey.h

```diff
@@ -444,6 +444,8 @@ enum class DispatchKey : uint16_t {
   Autograd,
   CompositeImplicitAutograd, // registered at
   // build/aten/src/ATen/RegisterCompositeImplicitAutograd.cpp
+  CompositeImplicitBatched, // registered at
+  // build/aten/src/ATen/RegisterCompositeImplicitBatched.cpp
```

Comment:
am I missing the piping here how is this file created? I just copied this from above